test/init.js

```diff
@@ -35,6 +35,7 @@ global.connectorCapabilities = {
   supportUpdateWithoutId: false,
   supportInclude: false,
   supportGeoPoint: false,
+  supportNullDataType: false
```

Comment:
nullDataValueExists
We are not defining if it support data type NULL but if null values of any data types exist in a particular database.